policy/modules/system/sysnetwork.te

```diff
@@ -380,6 +380,10 @@ ifdef(`distro_ubuntu',`
 	')
 ')
 
+optional_policy(`
+	container_read_runtime_files(ifconfig_t)
```

Comment:
How is this used? It seems unwise to allow ifconfig to read low integrity files.

Reply:
Allow me to double-check that this is still required and provide the AVC, but I found this was needed by netavark which will be invoked by podman to set up the container network. At some point during container startup it looked like the container's network data was being written to `/run`, and netavark would call `ip` to load it.

Reply:
The original AVC produced when I hit this is as follows:
```
type=PROCTITLE msg=audit(1679678391.757:560457): proctitle=69700061
type=SYSCALL msg=audit(1679678391.757:560457): arch=c000003e syscall=257 success=no exit=-13 a0=ffffff9c a1=74dfcdafcd10 a2=0 a3=0 items=0 ppid=3165222 pid=3568246 auid=1000 uid=0 gid=0 euid=0 suid=0 fsuid=0 egid=0 sgid=0 fsgid=0 tty=pts3 ses=30 comm="ip" exe="/usr/bin/ip" subj=staff_u:sysadm_r:ifconfig_t:s0 key=(null)
type=AVC msg=audit(1679678391.757:560457): avc:  denied  { read } for  pid=3568246 comm="ip" name="netns-6146bd2e-66b7-c06b-89d8-f992038d10bd" dev="tmpfs" ino=11056 scontext=staff_u:sysadm_r:ifconfig_t:s0 tcontext=system_u:object_r:container_runtime_t:s0 tclass=file permissive=0
```
However I have not been able to reproduce it, even after doing a `podman system reset` and restarting all containers on the host I saw this.

This might have instead been from trying to run `ip` directly and it trying to display information from other containers, but I wasn't able to reproduce this with that behavior either.

I'll drop this for now and we can revisit if it manifests again.